awsrds/rds_db_instance.go

```diff
@@ -101,6 +101,55 @@ func (r *RDSDBInstance) DescribeByTag(tagKey, tagValue string) ([]*DBInstanceDet
 	return dbInstanceDetails, nil
 }
 
+func (r *RDSDBInstance) GetTag(ID, tagKey string) (string, error) {
+
+	describeDBInstancesInput := &rds.DescribeDBInstancesInput{
+		DBInstanceIdentifier: aws.String(ID),
+	}
+
+	r.logger.Debug("get-tag", lager.Data{"input": describeDBInstancesInput})
+
+	dbInstances, err := r.rdssvc.DescribeDBInstances(describeDBInstancesInput)
```

Comment:
A: piotr says this is ok because `DescribeDBInstances` can return more instances. But I don't think it is fine, as I have only one ID and I expect to get a single instance here. Not 0, not more. So I'd call the variable `dbInstance` && check that it is exactly one. If not, panic.